### DISCUSSION TITLE
The simulator cannot run Stardew Valley Story.

### DISCUSSION THREAD

**Discussion by baby222666** [skyline-emu/skyline#1844] [Q&A]
The latest version of Stardew Valley Story will run on the screen.I hope it can be fixed.![Screenshot_2022-09-09-00-22-58-02.jpg](https://user-images.githubusercontent.com/110808515/189174731-1ec41cd4-9093-4345-87cc-77820f62b26b.jpg)![Screenshot_2022-09-09-00-22-59-85.jpg](https://user-images.githubusercontent.com/110808515/189174757-cd556829-0375-47ff-8f0d-b0130bbd20d2.jpg)Will be stuck in this interface.

**Comment by sans-pi** [comment]
Me encanta

**Comment by sans-pi** [comment]
No es justo asese cuenta
